app/controllers/merchants/items_controller.rb

```diff
@@ -30,6 +30,17 @@ def update
     end
   end
 
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+    @item = Item.new
+  end
+  
+  def create
+    merchant = Merchant.find(params[:merchant_id])
+    merchant.items.create(item_params)
```

Comment:
Future refactor id is to create a flash message for incomplete data. and have a test for sad path.